wiki/Mapping_and_Modding_Timeline/en.md

```diff
@@ -0,0 +1,458 @@
+    - The [Beatmap Nominator website](https://bn.mappersguild.com/) was published to automate many BN/NAT tasks.
+        - Through the website, users could apply to the BN at any time as opposed to the previous cycle-based application system. This resulted in Beatmap Nominators being added at irregular times instead of in large waves.
+        - Reports for inappropriate beatmaps and BN/NAT misconduct were shifted from a Google Form to the BN website.
+        - Statistics and evaluations for BN modding/nomination activity and proficiency shifted from painfully inefficient Google Spreadsheets to the BN website, hugely increasing productivity for Beatmap Nominator management. <!-- https://osu.ppy.sh/home/news/2019-05-05-bn-and-qat-restructure -->
+
+### August
+
+- **2019-08-05:** [Naxess](https://osu.ppy.sh/users/8129817) released the [Mapset Verifier](/wiki/Projects#modding) tool, automating a significant portion of beatmap checking processes for Beatmap Nominators.
+- **2019-08-06:** Beatmap Nominators became able to participate in BN application evaluations.
+    - BN input is visible to the NAT when trying to reach a consensus, though they are rarely used to sway a vote.
+    - The NAT often use BN input to help determine future NAT members. <!-- https://github.com/pishifat/qat/commit/a8245a4c6fefcfe9368ffc1eee7e13ad6f8bb6af -->
+
+### October
+
+- **2019-10-22:** [Ascendance](https://osu.ppy.sh/users/2931883) became a NAT leader after [Mao](https://osu.ppy.sh/users/2204515) stepped down.
+    - Because NAT leaders had often been passive towards negative decision making, Ascendance took an active controlling role and delivered bad news regarding NAT removals. <!-- source is discord logs -->
+
+**3217 beatmaps** were ranked in 2019.
+
+## 2020
+
+### February
+
+- **2020-02-07:** The first Elite Nominator profile badges were awarded to recognize continued exceptional performance from members of the BN team. 
+    - These were originally planned in the QAT Upheaval two years prior. <!-- https://osu.ppy.sh/home/news/2020-02-07-community-contributors-2019 -->
+- **2020-02-21:**  NAT leadership as a concept was discontinued as [Ascendance](https://osu.ppy.sh/users/2931883) stepped down as NAT leader.
+    - Tasks specific to NAT leaders were delegated to the whole team.
+- **2020-02-24:** The [Quality Assurance Helper](/wiki/People/The_Team/Beatmap_Nominators/General_Information#quality-assurance-helpers) system was transfered from Trello to the [Beatmap Nominator website](https://bn.mappersguild.com/).
+    - This change was made to account for Quality Assurance activity in Beatmap Nominator evaluations.
+
+### April
+
+- **2020-04-11:** The Mappers' Report <!-- TODO: stub --> led by [Feerum](https://osu.ppy.sh/users/4815717) was created to summarize mapping community events via front page news posts.
```

Comment:
if you have chance AND if there was anything significant in the sphere of modding and mapping, please document it

Reply:
nothing's coming to mind